src/services/smv_storage.py

```diff
@@ -31,6 +31,18 @@ def get_parties(self):
                 "party_id": item["pub_key"],
                 "twitter_handle": item["twitter_handle"],
                 "twitter_user_id": item["twitter_user_id"],
+                "last_modified": int(
+                    item["last_modified"]
+                    .replace(tzinfo=timezone.utc)
+                    .timestamp()
+                ),
+                # created might be missing in DB - use "last_modified" then
+                # TODO: remove usage of "last_modified" once no longer neede
```

Comment:
Now that the db has been nuked, can we remove the TODO and replace `item.get("created", item["last_modified"])` with `item["created"]` ?

Reply:
yes, we can :)